server/src/main/java/io/spine/server/entity/Repository.java

```diff
@@ -527,6 +529,23 @@ public void onDispatchCommand(Command command) {
             postSystem(systemCommand);
         }
 
+        /**
+         * Posts the {@link MarkCommandAsHandled} system command.
+         */
+        public void onCommandHandled(Command command) {
```

Comment:
It's still weird that an inner `protected` class has `public` methods. Let's discuss in person why we need that.